syftlib/src/main/java/org/openmined/syft/networking/requests/HttpAPI.kt

```diff
@@ -73,12 +75,27 @@ interface HttpAPI : CommunicationAPI {
         @Query("model_id") modelId: String
     ): Single<Response<ResponseBody>>
 
+    /**
+     * Calls `federated/authenticate` for authentication.
+     *
+     * @param authRequest Contains auth-token.
+     */
     @GET(AUTH_TYPE)
     override fun authenticate(authRequest: AuthenticationRequest): Single<AuthenticationResponse>
 
+    /**
+     * Calls `federated/cycle-request` for cycle details.
```

Comment:
Asks PyGrid to enter into a training cycle. The PyGrid responds with either accepting the worker by sending cycle accept response (attach see here) or cycle reject response (see)